src/index.css

```diff
@@ -30,17 +49,220 @@ nav ul {
 
 nav ul li {
   cursor: pointer;
+  padding-left: 5%;
   list-style: none;
 }
 
 .active-link {
   text-decoration: none;
+  color: black;
+}
+
+.logo {
+  color: #0590ff;
+}
+
+.user-image {
+  padding-left: 52%;
+  width: 40px;
+  height: 40px;
+}
+
+.each-item {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  justify-content: center;
+  align-content: center;
+  width: 50%;
+  height: 100%;
+  gap: 4px;
+  padding: 0 5% 0 5%;
+}
+
+.each-book-table {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 90%;
+  height: 8%;
+}
+
+.book-category {
+  font-size: 14px;
+  color: #bcbcbc;
+}
+
+.book-title {
+  color: #383838;
+  font-size: 28px;
+}
+
+.book-author {
+  color: #8dbce1;
+  font-size: 16px;
+}
+
+.book-button-holder {
+  display: flex;
+  gap: 10px;
+  padding-top: 3%;
+}
+
+.book-button {
+  color: #8dbce1;
+  padding: 0;
+  border: none;
+  background: none;
+}
+
+.inside-navbar {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.box {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.box .percent {
+  position: relative;
+  width: 150px;
+  height: 150px;
+}
+
+.box .percent svg {
+  position: relative;
+  width: 150px;
+  height: 150px;
+}
+
+.box .percent svg circle {
+  width: 150px;
+  height: 150px;
+  fill: none;
+  stroke-width: 10;
+  stroke: #000;
+  transform: translate(5px, 5px);
+  stroke-dasharray: 440;
+  stroke-dashoffset: 440;
+  stroke-linecap: round;
+}
+
+.box .percent .number {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #111;
+}
+
+.box .percent .number h2 {
+  font-size: 48px;
+}
+
+.box .percent .number h2 span {
+  font-size: 24px;
+}
+
+.box .text {
+  padding: 10px 0 0;
+  color: #999;
+  font-weight: 700;
+  letter-spacing: 1px;
+}
+
+.box .percent svg circle:nth-child(1) {
+  stroke-dashoffset: 0;
+  stroke: #f3f3f3;
+}
+
+.box .percent svg circle:nth-child(2) {
+  stroke-dashoffset: calc(440 - (440 * 60) / 100);
+  stroke: #03a9f4;
+}
+
+.item-all-table {
+  display: flex;
+  width: 100%;
+  height: 14rem;
+  flex-direction: row;
+  background-color: white;
+}
```

Comment:
- [ ] Please ensure that there is some space on the right of this div so that the content is well aligned as in the given zeplin design 👇 
- [ ] Please also add the vertical border on the right of the percentage completed
### Your Design
![image](https://user-images.githubusercontent.com/46839548/208250778-77293857-4bab-48d9-9fdf-117fd9e2ffb6.png)
### Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/208250831-d31e5af5-ee77-48e3-a539-a809bf20b884.png)